libs/ui/src/modals/screens/swap-route/swap-route.tsx

```diff
@@ -28,43 +27,40 @@ export const SwapRoute: FC = () => {
     setContentHeight(event.nativeEvent.layout.height - getCustomSize(3));
 
   return (
-    <ModalContainer screenTitle="Route">
-      <View style={styles.root}>
-        {loading ? (
-          <View style={styles.loader}>
-            <Loader size={LoaderSizeEnum.Large} />
-          </View>
-        ) : (
-          <ScrollView style={styles.content}>
-            <View onLayout={handleLayout}>
-              <VerticalLine height={contentHeight} />
-              <Row style={styles.fromToken}>
-                <Row>
-                  <View style={styles.triangle} />
-                  <Token symbol={params.fromToken.symbol} uri={params.fromToken?.thumbnailUri} forceHideTokenName />
-                </Row>
-
-                <Text style={styles.destination}>From</Text>
-              </Row>
+    <ModalActionContainer
+      screenTitle="Route"
+      buttonTitle="Close"
+      onPress={goBack}
+      contentContainerStyle={loading ? styles.contentContainerStyle : undefined}
+    >
+      {loading ? (
+        <View style={styles.loader}>
```

Comment:
Maybe we need to add a View wrapper inside the Loader component, we use this template many times in the project